managers.md

```diff
@@ -1,115 +1,107 @@
-# Engineering Management Competencies V2
+# Engineering Management Competencies
 
-The engineering manager ladder consists of three levels, Engineering Manager, Director and VP.
+The engineering manager ladder consists of four levels:
+Engineering Manager, Senior Engineering Manager, Director, and VP.
 
 ## Engineering Manager
 
 You are a frontline manager who is responsible for a functional area and leads a small team of up to six engineers.
```

Comment:
> small team of up to six engineers.

Do we want to put an upper boundary to the size of the team for an EM?  I'll rather have something like 'small to medium sized team'

Reply:
I agree.  We generally don't have huge teams anyway just as an organization precept.  I'll update.